packs/vsphere/tests/vsphere_base_action_test_case.py

```diff
@@ -57,3 +66,11 @@ def test_run_config_old(self):
     def test_run_config_new(self):
         action = self.get_action_instance(self.new_config)
         self.assertIsInstance(action, self.action_cls)
+
+    def test_run_config_old_partial(self):
+        self.assertRaises(KeyError, self.action_cls, self.old_config_partial)
```

Comment:
You can also use `assertRaisesRegexp` to also assert that a correct error message is thrown.